docs/source/03_plugin_structure.rst

```diff
@@ -0,0 +1,44 @@
+.. _plugin_structure-section:
+
+Plugins Structure
+=================
+
+As mentioned in :ref:`quick-start-section`, the |sdk| package has some utilities to help in the process to create
+a new plugin project and to create a plugin file.
+
+At any moment, it's possible to invoke the ``help`` command to list all commands available.
```

Comment:
I'm seeing a lot of `it's` around the whole documentation. Altough it is not wrong, for technical writing contractions like that are usually avoided. What do you think of doing a good global find+replace from `it's` to `it is`?

Reply:
Done